web/cobrands/northamptonshire/assets.js

```diff
@@ -618,4 +619,33 @@ function check_emergency() {
 }
 $(fixmystreet).on('report_new:category_change', check_emergency);
 
+function ncc_check_streetlights() {
+    var emergency_state = ncc_is_emergency_category();
+
+    if (emergency_state.relevant || fixmystreet.body_overrides.get_only_send() == 'Highways England') {
+        $('#ncc_streetlights').remove();
+        return;
+    }
+
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, northants_defaults.body) > -1;
+    var relevant_cat = $('#form_category').val() == 'Street lighting';
+    var relevant = relevant_body && relevant_cat;
+    var currently_shown = !!$('#ncc_streetlights').length;
+
+  console.log(relevant, currently_shown);
+    if (!relevant) {
+        $('#ncc_streetlights').remove();
+        enable_report_form();
```

Comment:
This worries me due to e.g. changing to a category that would need to hide it again (not on a road or something) but this running after, but as it's all hopefully refactoring I'm not too fussed!